[Spark GC] Log ranges and metaranges local disk storage

Under heavy loads (large repos, many deletions) Spark runs out of storage (disk).  Add logs to make it more obvious and easier to debug.

Typical error:
```
org.apache.hadoop.fs.FSError: java.io.IOException: No space left on device
	at org.apache.hadoop.fs.RawLocalFileSystem$LocalFSFileOutputStream.write(RawLocalFileSystem.java:500) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81) ~[?:?]
	at java.io.BufferedOutputStream.write(BufferedOutputStream.java:127) ~[?:?]
	at org.apache.hadoop.fs.FSDataOutputStream$PositionCache.write(FSDataOutputStream.java:63) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at java.io.DataOutputStream.write(DataOutputStream.java:112) ~[?:?]
	at org.apache.hadoop.io.IOUtils.copyBytes(IOUtils.java:98) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at org.apache.hadoop.io.IOUtils.copyBytes(IOUtils.java:69) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at org.apache.hadoop.io.IOUtils.copyBytes(IOUtils.java:133) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at org.apache.hadoop.fs.FileUtil.copy(FileUtil.java:490) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at org.apache.hadoop.fs.FileUtil.copy(FileUtil.java:430) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at org.apache.hadoop.fs.FileUtil.copy(FileUtil.java:368) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at org.apache.hadoop.fs.FileSystem.copyToLocalFile(FileSystem.java:2712) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	at io.treeverse.clients.LakeFSContext$.$anonfun$newRDD$5(LakeFSContext.scala:191) ~[lakefs-spark-client-assembly-0.18.0.jar:0.18.0]
	at scala.collection.Iterator$$anon$11.nextCur(Iterator.scala:486) ~[scala-library-2.12.17.jar:?]
	at scala.collection.Iterator$$anon$11.hasNext(Iterator.scala:492) ~[scala-library-2.12.17.jar:?]
	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460) ~[scala-library-2.12.17.jar:?]
	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460) ~[scala-library-2.12.17.jar:?]
	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.processNext(Unknown Source) ~[?:?]
	at org.apache.spark.sql.execution.BufferedRowIterator.hasNext(BufferedRowIterator.java:35) ~[spark-sql_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.sql.catalyst.expressions.GeneratedClass$GeneratedIteratorForCodegenStage3.hasNext(Unknown Source) ~[?:?]
	at org.apache.spark.sql.execution.WholeStageCodegenEvaluatorFactory$WholeStageCodegenPartitionEvaluator$$anon$1.hasNext(WholeStageCodegenEvaluatorFactory.scala:43) ~[spark-sql_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at scala.collection.Iterator$$anon$10.hasNext(Iterator.scala:460) ~[scala-library-2.12.17.jar:?]
	at org.apache.spark.shuffle.sort.BypassMergeSortShuffleWriter.write(BypassMergeSortShuffleWriter.java:168) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.shuffle.ShuffleWriteProcessor.doWrite(ShuffleWriteProcessor.scala:45) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.shuffle.ShuffleWriteProcessor.write(ShuffleWriteProcessor.scala:68) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:104) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.scheduler.ShuffleMapTask.runTask(ShuffleMapTask.scala:54) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.TaskContext.runTaskWithListeners(TaskContext.scala:161) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.scheduler.Task.run(Task.scala:143) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$4(Executor.scala:629) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally(SparkErrorUtils.scala:64) ~[spark-common-utils_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.util.SparkErrorUtils.tryWithSafeFinally$(SparkErrorUtils.scala:61) ~[spark-common-utils_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:95) ~[spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:632) [spark-core_2.12-3.5.0-amzn-0.jar:3.5.0-amzn-0]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:840) [?:?]
Caused by: java.io.IOException: No space left on device
	at java.io.FileOutputStream.writeBytes(Native Method) ~[?:?]
	at java.io.FileOutputStream.write(FileOutputStream.java:349) ~[?:?]
	at org.apache.hadoop.fs.RawLocalFileSystem$LocalFSFileOutputStream.write(RawLocalFileSystem.java:496) ~[hadoop-client-api-3.3.6-amzn-2.jar:?]
	... 36 more
```

--- a/clients/spark/src/main/scala/io/treeverse/clients/SSTableReader.scala
+++ b/clients/spark/src/main/scala/io/treeverse/clients/SSTableReader.scala
@@ -82,7 +82,9 @@ object SSTableReader {
       own: Boolean = true
   ): SSTableReader[RangeData] = {
     val localFile: File = copyToLocal(configuration, metaRangeURL)
-    logger.info(s"local ${localFile.getAbsolutePath} (metarange ${metaRangeURL}) copied, ${localFile.length()} bytes")
+    logger.info(
+      s"local ${localFile.getAbsolutePath} (metarange ${metaRangeURL}) copied, ${localFile.length()} bytes"
+    )
     val ret = new SSTableReader(localFile, RangeData.messageCompanion, own)
     Option(TaskContext.get()).foreach(_.addTaskCompletionListener[Unit](_ => ret.close()))
     ret
@@ -94,7 +96,9 @@ object SSTableReader {
       own: Boolean = true
   ): SSTableReader[Entry] = {
     val localFile: File = copyToLocal(configuration, rangeURL)
-    logger.info(s"local ${localFile.getAbsolutePath} (range ${rangeURL}) copied, ${localFile.length()} bytes")
+    logger.info(
+      s"local ${localFile.getAbsolutePath} (range ${rangeURL}) copied, ${localFile.length()} bytes"
+    )
     val ret = new SSTableReader(localFile, Entry.messageCompanion, own)
     Option(TaskContext.get()).foreach(_.addTaskCompletionListener[Unit](_ => ret.close()))
     ret

